libraries/plugins/apis/condenser_api/include/steemit/plugins/condenser_api/condenser_api_plugin.hpp

```diff
@@ -11,7 +12,7 @@ using namespace appbase;
 class condenser_api_plugin : public appbase::plugin< condenser_api_plugin >
 {
 public:
-   APPBASE_PLUGIN_REQUIRES( (steemit::plugins::json_rpc::json_rpc_plugin)(steemit::plugins::database_api::database_api_plugin) )
+   APPBASE_PLUGIN_REQUIRES( (steemit::plugins::json_rpc::json_rpc_plugin)(steemit::plugins::database_api::database_api_plugin)(steemit::plugins::block_api::block_api_plugin) )
```

Comment:
Remove the dependency on block_api_plugin. The point of `api_startup` is to dynamically detect which APIs are loaded without needing a strict dependency. One of the design goals of moving `get_block` to its own API is to disable it if it becomes too expensive of a call.